tests/ci/digest_helper.py

```diff
@@ -78,3 +75,9 @@ def digest_script(path_str: str) -> HASH:
         logger.warning("The modules size has changed, retry calculating digest")
         return digest_script(path_str)
     return md5_hash
+
+
+def digest_string(string: str) -> str:
```

Comment:
It's trivial. Do we need it?

Reply:
i think all the methods for making a digest should be in one place. if we use digest_helper why should we care about hashing algorithms and etc.